parm/archive/gefs_atmos.yaml.j2

```diff
@@ -0,0 +1,58 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".t" + cycle_HH + "z." %}
+
+gefs_atmos:
+    name: "GEFS_ATMOS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_atmos.tar"
+    required:
+#select mem%03d and ensstat files required
+{% set membr = ["ensstat"] %}
+{% for mem_nm in range(0, NMEM_ENS + 1) %}
+    {% do membr.append("mem" ~ '%03d' % mem_nm ) %}
+{% endfor %}
+
+#need some logic here to select FHOUT_HF_GFS or FHOUT_GFS
+#for the time being using FHOUT_GFS
+{% set fcst_freq = FHOUT_GFS %}
+{% set max_hr = FHMAX_GFS %}
+
+{% if REPLAY_ICS == "YES" %}
+    {% set ofst_hr = OFFSET_START_HOUR %}
+{% else %}
+    {% set ofst_hr = FHMIN_GFS %}
+{% endif %}
+
+{% for mem in membr %}
+    {% for res in ['0p25', '0p50', '1p00'] %}
+        {% set tmpl_dict = ({ '${ROTDIR}':ROTDIR,
+                              '${RUN}':RUN,
+                              '${YMD}':cycle_YMD,
+                              '${HH}':cycle_HH,
+                              '${GRID}':res,
+                              '${MEMDIR}':mem }) %}
+
+        {% set COMIN_ATMOS_GRIB = COM_ATMOS_GRIB_GRID_TMPL | replace_tmpl(tmpl_dict) %}
+
+        # Select pgrb and grib files to copy to the atardir
+        {% if RUN == "gefs" %}
```

Comment:
I think this `if` is unnecessary; this jinja would not be used if this were not gefs.